packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -2429,6 +2429,56 @@ HTTP/1.1 200 OK
 Content-Type: application/json
 ```
 
+### Clear Current User Sessions
+
+Clears all Okta sessions for the currently logged in user. This forces the user to authenticate on the next operation. Use this method in a browser-based application. 
```

Comment:
calling this API without any params will not force the user to authenticate again, since the default value for `keepCurrent` is true...
I think we should change it to be more explicit about it.. what about:
```
Clears Okta sessions for the currently logged in user. By default, the current session remains active. Use this method in a browser-based application. 
```